bin/gwdetchar-scattering

```diff
@@ -164,7 +164,7 @@ if args.gpsstart >= 1230336018:  # Jan 1 2019
     read_kw = {
         "columns": names,
         "selection": [
-            "frequency < {}".format(args.frequency_threshold * 2),
+            "{0} < frequency < {1}".format(10*1, args.frequency_threshold * 4),
```

Comment:
@siddharth101, can you remove the `*1`?

```suggestion
            "{0} < frequency < {1}".format(10, args.frequency_threshold * 4),
```

I have the same comment for line 186 below.